File: Ui-TravelApp/code/app/src/main/java/com/doanchung/ui_travelapp/Activity/IntroActivity.java
package com.doanchung.ui_travelapp.Activity;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.doanchung.ui_travelapp.databinding.ActivityIntroBinding;

public class IntroActivity extends AppCompatActivity {

    ActivityIntroBinding binding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityIntroBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.startBtn.setOnClickListener(
                v -> startActivity(
                        new Intent(IntroActivity.this, MainActivity.class)
                )
        );

    }
}

File: ShoesShopEcommerce/app/src/main/java/com/doanchung/shoes_shop_ecommerce/activity/IntroActivity.kt
package com.doanchung.shoes_shop_ecommerce.activity

import android.content.Intent
import android.os.Bundle
import android.widget.Toast
import com.doanchung.shoes_shop_ecommerce.R
import com.doanchung.shoes_shop_ecommerce.databinding.ActivityIntroBinding
class IntroActivity : BaseActivity() {
    private lateinit var binding: ActivityIntroBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityIntroBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.startBtn.setOnClickListener {
            Toast.makeText(this@IntroActivity, "Button clicked!", Toast.LENGTH_SHORT).show()

            startActivity(
                Intent(
                    this@IntroActivity, MainActivity::class.java
                )
            )
        }
    }
}